pkg/asset/installconfig/aws/permissions.go

```diff
@@ -201,6 +202,7 @@ var permissions = map[PermissionGroup][]string{
 		"iam:ListInstanceProfiles",
 		"iam:ListRolePolicies",
 		"iam:ListUserPolicies",
+		"s3:DeleteBucket",
```

Comment:
Do we need this one if its a terraform install? This is covered by the `PermissionDeleteIgnitionObjects`. I could see this one being `"s3:DeleteObject"` since it is for a base capa install.

Reply:
We do (for both) because we delete the bucket during cluster destroy.